en/deploy-on-alibaba-cloud.md

```diff
@@ -237,12 +264,17 @@ This may take a while to complete. You can watch the process using the following
 kubectl get pods --namespace ${namespace} -o wide --watch
 ```
 
-## Scale
+## Scale out TiDB cluster
 
-To scale the TiDB cluster, modify `tikv_count` or `tidb_count` in the `terraform.tfvars` file, and then run `terraform apply` to scale out the number of nodes for the corresponding components.
+To scale out the TiDB cluster, modify `tikv_count`, `tiflash_count` or `tidb_count` in the `terraform.tfvars` file, and then run `terraform apply` to scale out the number of nodes for the corresponding components.
 
 After the nodes scale out, modify the `replicas` of the corresponding components by running `kubectl --kubeconfig credentials/kubeconfig edit tc ${tidb_cluster_name} -n ${namespace}`.
 
+> **Note:**
+>
+> - Because it is impossible to determine which node is taken offline during the scale-in process, the scaling in of TiDB clusters is currently not supported.
```

Comment:
```suggestion
> - Because it is impossible to determine which node is taken offline during the scale-in process, the scale-in of TiDB clusters is currently not supported.
```